modules/rest-util/src/blaze/middleware/fhir/output.clj

```diff
@@ -1,5 +1,7 @@
 (ns blaze.middleware.fhir.output
-  "JSON/XML serialization middleware."
+  "Resources serialization middleware.
+
+  Currently supports JSON, XML, and binary."
```

Comment:
"binary" isn't a common resource serialization format like JSON or XML. It's rather a special serialization of the Binary resource only. I would write: 

FHIR JSON/XML serialization middleware.

In addition to the common JSOM/XML serializations, `wrap-binary-output` will
output the binary data of the Binary resource if requested.

Reply:
I will move all implementation-specific mentions to the specific function's clojuredoc string, so we don't have to keep changing it here.